orchagent/aclorch.cpp

```diff
@@ -1042,6 +1046,14 @@ bool AclRuleL3V6::validateAddMatch(string attr_name, string attr_value)
         return false;
     }
 
+    // NOTE: For backwards compatibility, allow users to substitute IP_PROTOCOL for NEXT_HEADER.
```

Comment:
I think you can have a TODO comment here and remove the commented out code.

Reply:
Done!